api/Navigator.json

```diff
@@ -3072,6 +3063,1638 @@
               "deprecated": false
             }
           }
+        },
+        "scheme_parameter_cabal": {
```

Comment:
FYI @ddbeck This is the newly added non-standard chromium extension for "cabal" added in https://chromestatus.com/feature/4776602869170176 - just added to ease review - these are all the same versions, only the name of the subfeature and the description are changed.